src/components/Calls/CallNotification.js

```diff
@@ -1,30 +1,36 @@
 import { h } from 'preact';
 import { useState } from 'preact/compat';
 
+import { Livechat } from '../../api';
 import I18n from '../../i18n';
 import PhoneAccept from '../../icons/phone.svg';
 import PhoneDecline from '../../icons/phoneOff.svg';
 import constants from '../../lib/constants';
 import { Avatar } from '../Avatar';
 import { Button } from '../Button';
 import { createClassName, getAvatarUrl } from '../helpers';
+import { CallIframe } from './CallIFrame';
 import styles from './styles.scss';
 
 
-export const CallNotification = ({ callProvider, callerUsername, url, dispatch } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined }) => {
+export const CallNotification = ({ callProvider, callerUsername, url, dispatch, rid } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined, rid: undefined }) => {
 	const [show, setShow] = useState(!!callProvider && !!callerUsername && !!dispatch && !!url);
+	const [isIframe, setIframe] = useState(false);
+	const { token } = Livechat.credentials;
+	const msg = 'JOIN_CALL_MSG';
+
 
 	const acceptClick = async () => {
 		setShow(!{ show });
-
+		await Livechat.sendMessage({ token, rid, msg });
```

Comment:
This isn't a good UI/UX. The message which you're sending here is visible to the agent as well but its of no use to them, see here
![image](https://user-images.githubusercontent.com/34130764/126774536-a5c1e846-fc87-4329-bd41-dd1fce5b6f66.png)